packages/utils/skpm-config.js

```diff
@@ -1,10 +1,10 @@
 const extractRepository = require('./extract-repository')
 
-module.exports = function getSkpmConfig(packageJSON) {
+module.exports = function getSkpmConfig(packageJSON, argv) {
   const skpmConfig = packageJSON.skpm || {}
   return {
     main: skpmConfig.main || packageJSON.main,
-    manifest: skpmConfig.manifest || packageJSON.manifest,
+    manifest: argv.manifest || skpmConfig.manifest || packageJSON.manifest,
```

Comment:
we need to check that `argv` is not `undefined` here, this method is used in some other places which won't provide `argv` yet

Reply:
not sure, we might want to allow this argument in other commands as well. And potentially overwrite `main`? so we will keep everything in one place and commands don't have to worry about arguments, overwrites and such. They just use `skpmConfig`.